WMCO 10.16.1 release notes for OCP 4.16 (#80455)
openshift/openshift-docs: windows_containers/windows-containers-release-notes-10-16-x.adoc

If you do not have this additional Red{nbsp}Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
endif::openshift-origin[]

[id="wmco-10-16-1"]
== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.16.1

This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.16.1 were released in link:https://access.redhat.com/errata/RHBA-TBD[RHBA-TBD].

[id="wmco-10-16-1-bug-fixes"]
=== Bug fixes

> * Previously, if a Windows VM had its PowerShell `ExecutionPolicy` set to `Restricted`, the Windows Instance Config Daemon (WICD) was unable to run the commands on that VM that are necessary for launching Windows nodes. With this fix, the WICD now bypasses the execution policy on the VM when running PowerShell commands. As a result, the WICD can create Windows nodes on the VM as expected. (link:https://issues.redhat.com/browse/OCPBUGS-37609[*OCPBUGS-37609*])

Reviewer: ```suggestion
* Previously, if a Windows VM had its PowerShell `ExecutionPolicy` set to `Restricted`, the Windows Instance Config Daemon (WICD) could not run the commands on that VM that are necessary for launching Windows nodes. With this fix, the WICD now bypasses the execution policy on the VM when running PowerShell commands. As a result, the WICD can create Windows nodes on the VM as expected. (link:https://issues.redhat.com/browse/OCPBUGS-37609[*OCPBUGS-37609*])
```
- Tiny suggestion for simplicity :)

- Also, IBM Style Word Usage warns about the word _launch_, and advises using _start_ or _open_ unless _launch_ is a standard part of a product or component's terminology.
